### PR TITLE
search results and query parameter 

### PR DESCRIPTION
**Pull request by NzingaHolloman**

### CODE REVIEW

File: src/main/webapp/SearchResults.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="zxx">
```
Comment (josueetcom) [inline src/main/webapp/SearchResults.html]: I'm still confused about the lang `zxx`

File: src/main/webapp/SearchResults.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styleNzinga.css" />
+    <script src="scriptNzinga.js"></script>
```
Comment (josueetcom) [inline src/main/webapp/SearchResults.html]: For consistency you may want to rename these files as `SearchResults.css` and `SearchResults.js` so it's easier to spot which files correspond with this page

File: src/main/webapp/scriptNzinga.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+  const city3Container = document.getElementById('city3');
+  city3Container.innerText = 'Jeju-do';
+
+  const cityDesc3Container = document.getElementById('cityDesc3');
+  cityDesc3Container.innerText = 'Jeju province encompasses the ' +
+      'South Korean island of Jeju in the Korea Strait. Its known for' +
+      'its beach resorts and volcanic landscape of craters and cavelike' +
+      ' lava tubes.';
+}
```
Comment (josueetcom) [inline src/main/webapp/scriptNzinga.js:128]: Please refactor this logic as it's pretty repetitive across all three functions. You might consider something like this:
```javascript
function malaysiaResults() {
  const headingContainer = document.getElementById('Heading');
  headingContainer.innerText = 'Malaysia';
  
  const city1 = {
      name: "Kuala Lumpur",
      description: "Description for Kuala Lumpur",
      id: 101,
  }
  const city2 = { /* city2 info */ }
  const city3 = { /* city3 info */ }
  displayResults(city1, city2, city3);
}

// similar brazilResults() and southKoreaResults() functions

function displayResults(city1, city2, city3) {
  // populate city1 info
  // populate city2 info
  // populate city3 info
}
```

Or even better (either in this PR or the next one), you could make `displayResults()` generic enough to accept any number of results and create the necessary HTML elements:
```javascript
function malaysiaResults() {
  const headingContainer = document.getElementById('Heading');
  headingContainer.innerText = 'Malaysia';
  
  const results = [{
      name: "Kuala Lumpur",
      description: "Description for Kuala Lumpur",
      id: 101,
  },
  // any additional cities
  ]

  displayResults(results);
}

// similar brazilResults() and southKoreaResults() functions

function displayResults(results) {
  for (let result in results) {
    // Create a row div if necessary or grab last row
    // Create a column div
    // Create a card div
    // Create the elements that go inside the card div
    // populate city information
    // append the information elements to the card div
    // append the card div to the column div
    // append the column div to the row div
  }
}
```
The latter option is closer to what's needed for the final product. For additional help I suggest taking a look at your comments feature from week 2 and also taking a look at the [Todo List](https://github.com/google/software-product-sprint/blob/master/walkthroughs/week-2-server/examples/todo-list/src/main/webapp/script.js) example from week 2.

**Review by josueetcom — CHANGES_REQUESTED** [review]

**Review by gferioli0418 — COMMENTED** [review]
other than the comments made by Josue it looks good to me

**Review by josueetcom — APPROVED** [review]
I think you need to `git rm src/main/webapp/scriptNzinga.js` so that it's no longer tracked (you can do that in a follow up PR). Otherwise looks good to me! I will submit this as is.

**Pull request merged**